config/systemConfig/python.yaml

```diff
@@ -0,0 +1,9 @@
+systemConfig:
+  - key: apiHttpLibraries
+    value: (?i)(request|aiohttp|treq|grequests|urllib|http|uplink|httoop|flask_restful|tornado.httpclient|pycurl|bs4|.*(HttpClient)).*
```

Comment:
Not sure if we can make api calls using `bs4`. can we get this checked ?

Reply:
@dbMundada @hiteshbedre this is the python api libraries, have moved them to privado from privado-core

Reply:
It is a web scrapping library from which XML or HTML content can be obtained. Let's keep it for now

Reply:
Right, @khemrajrathore  bs4 used to scrape the websites & get the data.